ruby/README.md

```diff
@@ -2,8 +2,12 @@
 
 [Sample 1](sample_1.rb) [Sample 2](sample_2.rb)
 
+> [!TIP]
```

Comment:
This renders as
<img width="1040" alt="image" src="https://github.com/user-attachments/assets/9a9e6f5e-f4dd-4337-b855-ef96d20e7d40" />